Formula/qmk.rb

```diff
@@ -25,6 +25,7 @@ class Qmk < Formula
   depends_on "pillow"
   depends_on "python"
   depends_on "teensy_loader_cli"
+  depends_on "pkg-config"
```

Comment:
This should be moved above avrdude, and set to a build dependency. Please also set the `revision` to 1.